retail/solutions/frontline-worker/README.md

```diff
@@ -0,0 +1,145 @@
+# Real-time store communications and collaboration
+
+## Table of Contents
+
+- [Microsoft Teams Integration with D356 Commerce](#microsoft-teams-integration-with-d365-commerce)
+  - [Table of Contents](#table-of-contents)
+
+
+## Microsoft Teams Integration with D356 Commerce
+
+[Dynamics 365 Commerce is integrating with Microsoft Teams](https://docs.microsoft.com/en-us/dynamics365/teams-integration/teams-integration) to help customers and their employees improve productivity by synchronizing task management between the two applications. The seamless task management that Commerce and Teams integration provides let store managers and employees create task lists, assign tasks to multiple stores, and track the status of tasks across stores, from either application.
```

Comment:
Please remove "/en-us" from the hyperlink. The hyperlink here will direct readers to EN-US version of the webpage.

Reply:
Change "Dynamics 365 Commerce is integrating with Microsoft Teams" to "Dynamics 365 Commerce is integrated with Microsoft Teams".

Reply:
Reword this sentence ? "The seamless task management that Commerce and Teams integration provides let store managers" to "Seamless integration between D365 Commerce and Teams simplifies task management, and it enables store managers and employees to create task lists; assign task lists to stores; and also track the status of various tasks across stores."